keel-artifact/src/main/kotlin/com/netflix/spinnaker/keel/artifacts/ArtifactListener.kt

```diff
@@ -52,19 +52,25 @@ class ArtifactListener(
       .artifacts
       .filter { it.type.toLowerCase() in artifactTypeNames }
       .forEach { artifact ->
```

Comment:
"registered" means that we are saving versions for that artifact name + type. It's purposefully separate from the delivery config so that multiple apps can care about the same debian package or docker image